test/jdk/java/nio/channels/AsyncCloseAndInterrupt.java

```diff
@@ -134,10 +134,15 @@ private static void initFile() throws Exception {
                 throw new IOException("Cannot delete existing fifo " + fifoFile);
         }
         Process p = Runtime.getRuntime().exec("mkfifo " + fifoFile);
-        if (p.waitFor() != 0)
-            throw new IOException("Error creating fifo");
+        if (p.waitFor() != 0) {
```

Comment:
use ProcessTools.executeProcess for this; it has facilities to check the exit code and dump process output on failure.

Reply:
See version as of commit 79b3c50e7a32472bcfc5cd8026254257a0d210b0.